src/main/kotlin/net/irisshaders/lilybot/extensions/util/ModUtilities.kt

```diff
@@ -43,9 +43,15 @@ class ModUtilities : Extension() {
 						respond { content = "**Error:** You do not have the `Moderate Members` permission" }
 						return@action
 					}
-					val actionLogId = getConfigPrivateResponse("modActionLog") ?: return@action
-
-					val actionLog = guild?.getChannel(actionLogId) as GuildMessageChannelBehavior
+					val config = DatabaseHelper.getConfig(guild!!.id)
+					if (config == null) {
```

Comment:
Use an elvis operator instead?

Reply:
Wait why are there no checks on this command?

Reply:
Explained on discord